src/flags.ts

```diff
@@ -43,11 +44,25 @@ const defaultValues = {
       flags[FlagNames.GIT_CHANGE_TARGET] = process.env.CHANGE_TARGET
     }
   },
+  [FlagNames.BUILD_SCRIPT]: async (flags: any) => {
```

Comment:
Yes, but we are starting to identify pattern and bundle them as archetype, so that they don't need to provide the build/deployment scripts (or OpenShift templates) anymore.

Reply:
The build/deployment script  path is based on providing an `archetype`. If an archetype is provided, it looks into the bundled archetype files, if not, it looks into the current directory. They were initially configurable as a way to support testing, and now we actually a need.